src/cmbuild/README.txt

```diff
@@ -1 +0,0 @@
-This directory is provided for convenience to do CMake builds...
```

Comment:
To keep the `cmbuild` directory, you will need to add a `.gitignore` as in the cmake PR.  FYI, I am wondering about the possibility of changing this convention to one that uses a `build` directory in the root of the repo, because that seems to be a convention that people use, but we should discuss first.

Reply:
The idea of moving `build` sounds good!

Sorry for the mistake, file restored in [194b0af](https://github.com/hypre-space/hypre/pull/1190/commits/194b0af98fdf6db4ccf7164370f08ce9590ee788)